django_lightweight_queue/urls.py

```diff
@@ -1,9 +1,9 @@
-from django.conf.urls import url
+from django.urls import re_path
 
 from . import views
 
 app_name = 'django_lightweight_queue'
 
 urlpatterns = (
-    url(r'^debug/django-lightweight-queue/debug-run$', views.debug_run, name='debug-run'),
+    re_path(r'^debug/django-lightweight-queue/debug-run$', views.debug_run, name='debug-run'),
```

Comment:
I think this could use `path` instead as there's not actually any pattern here?
```suggestion
    path('debug/django-lightweight-queue/debug-run', views.debug_run, name='debug-run'),
```
(obviously the import would need updating too)

Reply:
seems reasonable to me!